src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,223 @@
+// #region imports
+import classNames from 'classnames';
+import React, { useEffect, useRef, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { getUserById } from '../../services/user';
+import usersFromServer from '../../api/users';
+// #endregion
+
+type Props = {
+  onSubmit: (todo: Todo) => void;
+  onReset?: () => void;
+  todo?: Todo;
+  body: string;
```

Comment:
The `body` prop is defined in the `Props` type but is not used in the component. Consider using it or removing it if it's unnecessary.